### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by grosser**
@zendesk/samson 

### Risks
 - None

### CODE REVIEW

File: app/views/shared/_output.html.erb [hunk 1]
```diff
@@ -8,13 +8,13 @@
       </div>
 
       <button class="btn btn-default only-finished" id="output-grow-toggle">Expand</button>
-      <%= link_to "Download log", project_deploy_path(@project, @deploy, format: :text), class: "btn btn-primary only-finished" %>
+      <%= link_to "Download log", project_deploy_path(@project, deployable, format: :text), class: "btn btn-primary only-finished" %>
```
Comment (sandlerr) [inline app/views/shared/_output.html.erb]: does `@deploy` still need to be an ivar?

Comment (grosser) [inline app/views/shared/_output.html.erb]: no since this redirect to either the job or the deploy -> deployable

Comment (sandlerr) [inline app/views/shared/_output.html.erb]: nvm, still using the ivar in https://github.com/zendesk/samson/pull/784/files#diff-00955b7a831930544081954ec53bf6bfL4 (was thinking we may be able to make it a local if it was assigned in that method)

**Comment by sandlerr** [comment]
:+1:

File: app/views/jobs/_output.html.erb [hunk 2]
```diff
@@ -1,20 +0,0 @@
...
-      <%= link_to "Download log", project_job_path(@project, job, format: :text), class: "btn btn-primary only-finished" %>
-
-      <% if job.can_be_stopped_by?(current_user) %>
-        <%= stop_button(deploy: job, class: "btn btn-danger only-active") %>
-      <% end %>
-    </div>
-  </div>
-
-  <%= render partial: "jobs/log", locals: { job: job } %>
-</div>
```
Comment (jonmoter) [inline app/views/jobs/_output.html.erb]: This file was being used on the builds/show view.  :cry:

**Pull request merged**